lib/python/picongpu/input/parameters.py

```diff
@@ -208,7 +241,8 @@ class LogScaledParameter(Parameter):
     """
 
     def __init__(self, name, ptype, unit, default, slider_min, slider_max,
```

Comment:
LogScaledParameter should actually inherit from UiParameter. I commited that wrong and it needs to be fixed.